### PR TITLE
Handle case for az cmd for windows file diag script

### PR DESCRIPTION
**Pull request by bharathsm-ms**
Handle a case when Get-AzStorageFileServiceProperty returns NULL for some properties

### CODE REVIEW

File: AzFileDiagnostics/Windows/AzFileDiagnostics.ps1 [hunk 1]
```diff
@@ -1427,6 +1427,11 @@ function GetServerProtocolSettings($PSStorageAccount) {
 
     $smbProtocolSettings = Get-AzStorageFileServiceProperty -StorageAccount $PSStorageAccount
 
+    if($null -eq $smbProtocolSettings.ProtocolSettings) {
```
Comment (sprasad-microsoft) [inline AzFileDiagnostics/Windows/AzFileDiagnostics.ps1:1430]: Is there a chance that ProtocolSettings is non-null, but ProtocolSettings.Smb is null?

Comment (bharathsm-ms) [reply AzFileDiagnostics/Windows/AzFileDiagnostics.ps1:1430]: yes. if nobody every changed anything in protocol setting page of azure portal, we get it as NULL.

Comment (sprasad-microsoft) [reply AzFileDiagnostics/Windows/AzFileDiagnostics.ps1:1430]: Ok. Please also check if you're handling the same in Linux script too

Comment (bharathsm-ms) [reply AzFileDiagnostics/Windows/AzFileDiagnostics.ps1:1430]: Yes. It being handled in Linux script.

**Pull request merged**